terraform/modules/daos_server/variables.tf

```diff
@@ -142,13 +142,28 @@ variable "preemptible" {
   type        = string
 }
 
+variable "pool_cont_create" {
```

Comment:
This seems like an orthogonal knob with server_pools, right?
If server_pools = [] then this means false, if server_pools is populated, then this should be automatically true, or am I missing something? Why do you need this variable?